src/client/client-wrapper.ts

```diff
@@ -1,237 +1,31 @@
 import * as grpc from 'grpc';
+import { BasicInteractionAware, DomAware, ResponseAware } from './mixins';
```

Comment:
I like! 😄

Reply:
I just remembered about `barrels` now. Should have implemented in `salesforce` too